inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -232,7 +232,7 @@ ie_developer_export_targets(${TARGET_NAME} ${TARGET_NAME}_plugin_api xbyak)
 
 list(APPEND core_components ngraph)
 
-if(THREADING STREQUAL "TBB" OR THREADING STREQUAL "TBB_AUTO")
+if((THREADING STREQUAL "TBB" OR THREADING STREQUAL "TBB_AUTO") AND TBBROOT STREQUAL "")
```

Comment:
I suppose `TBBROOT` is always not empty. See https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/cmake/dependencies.cmake#L171

Reply:
I stand corrected. Evaluating `TBBROOT` here is indeed not the correct condition. I'm not yet sure what the real solution is.

Reply:
You can compare `TBBROOT` with `${TEMP}` (`if(${var_name} MATCHES ${TEMP})`). The OpenVINO CMake downloads artifacts to `${TEMP}` directory, so if the `TBBROOT` points to this directory it is downloaded artifact. `reset_deps_cache` uses the same principle.

Reply:
Thank you for the suggestion. I tested it and updated the PR.